app/forms/concerns/steps/ownership_confirmation.rb

```diff
@@ -0,0 +1,37 @@
+module Steps
+  module OwnershipConfirmation
```

Comment:
Is the form intentionally not populating the ownership checkbox when the user returns to that page having already checked it?

Reply:
Seen the spec, yes it is. Ignore 👍